prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -97,16 +97,28 @@ presubmits:
         command:
         - ./test/presubmit-tests-tfx.sh
 
+<<<<<<< HEAD
```

Comment:
did not resolve merge conflict

Reply:
Seemed like a git push issue. Pushed again, should be resolved and rebased correctly now. Could you please check?